frontend/src/components/LandingPage.vue

```diff
@@ -664,15 +670,19 @@ export default {
       this.setIsRenewal(false);
       this.$router.push(
         pcfUrl(
-          this.organizationProviderType === 'GROUP' ? PATHS.CCOF_GROUP_ORG : PATHS.CCOF_FAMILY_ORG,
+          this.organizationProviderType === ORGANIZATION_PROVIDER_TYPES.GROUP
+            ? PATHS.CCOF_GROUP_ORG
+            : PATHS.CCOF_FAMILY_ORG,
           this.programYearId,
         ),
       );
     },
     goToCCOFOrganizationInfo() {
       this.$router.push(
         pcfUrl(
-          this.organizationProviderType === 'GROUP' ? PATHS.CCOF_GROUP_ORG : PATHS.CCOF_FAMILY_ORG,
+          this.organizationProviderType === ORGANIZATION_PROVIDER_TYPES.GROUP
+            ? PATHS.CCOF_GROUP_ORG
+            : PATHS.CCOF_FAMILY_ORG,
```

Comment:
Not a big deal but logic is repeated here.

Reply:
true enough - I merged these two together. It is safe because if you are viewing CCOF - you are always going to be in an application state of NEW